src/extension/configuration.h

```diff
@@ -41,7 +49,6 @@ extern bool runtime_config_first_init;
     SYSCFG(BOOL, DD_APPSEC_TESTING, "false")                                                                    \
     SYSCFG(BOOL, DD_APPSEC_TESTING_ABORT_RINIT, "false")                                                        \
     SYSCFG(BOOL, DD_APPSEC_TESTING_RAW_BODY, "false")                                                           \
-    SYSCFG(CUSTOM(STRING), DD_APPSEC_IPHEADER, "", .parser = dd_parse_ipheader_config)                          \
```

Comment:
I think the existing code could follow the same approach for `DD_TRACE_CLIENT_IP_HEADER`, i.e. using a parsing function. Any particular reason why the new implementation uses a different approach?

Reply:
Done